Project.toml

```diff
@@ -50,6 +51,7 @@ julia = "1.10"
 pocl_jll = "7"
 
 [extras]
+StaticArrays = "90137ffa-7385-5640-81b9-e52037218182"
 EnzymeCore = "f151be2c-9106-41f4-ab19-57ee4f262869"
 LinearAlgebra = "37e2e46d-f89d-539d-b4ee-838fcccc9c8e"
 SparseArrays = "2f01184e-e22b-5df5-ae63-d93ebab69eaf"
```

Comment:
```suggestion
```
Since test/Porject.toml exists, I think this can be deleted and the `StaticArrays` entry added to that file instead